vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/video/ScreenRecorderStopper.kt

```diff
@@ -55,8 +64,8 @@ internal class ScreenRecorderStopper(private val deviceInterface: IDevice) {
 
     companion object {
         private val logger = MarathonLogging.logger("ScreenRecorderStopper")
-        private const val SCREEN_RECORD_KILL_ATTEMPTS = 100
-        private const val PAUSE_BETWEEN_RECORDER_PROCESS_KILL = 50
+        private const val SCREEN_RECORD_KILL_ATTEMPTS = 5
+        private const val PAUSE_BETWEEN_RECORDER_PROCESS_KILL = 300
```

Comment:
Can you please mention as a comment in the code that we've empirically found this to be the only way to fix the broken video files? These delays of random millis are really confusing otherwise. And a link to the issue would also be nice.